README.md

```diff
@@ -88,43 +88,9 @@ editor of your choice and edit the `username` and `authtoken` attribute under th
 
 <strong>Configure config.yml File</strong>
 
-*Note*: Below is only a brief explanation of some attributes in `config.yml`. <strong>Please use the link [here](https://github.com/Sage-Bionetworks/schematic/blob/develop/config.yml) to get the latest version of `config.yml` in `develop` branch</strong>.
-
-Description of `config.yml` attributes
-
-    definitions:
-        synapse_config: "~/path/to/.synapseConfig"
-        service_acct_creds: "~/path/to/service_account_creds.json"
-
-    synapse:
-        master_fileview: "syn23643253" # fileview of project with datasets on Synapse
-        manifest_folder: "~/path/to/manifest_folder/" # manifests will be downloaded to this folder
-        manifest_basename: "filename" # base name of the manifest file in the project dataset, without extension
-        service_acct_creds: "syn25171627" # synapse ID of service_account_creds.json file
-
-    manifest:
-        title: "example" # title of metadata manifest file
-        # to make all manifests enter only 'all manifests'
-        data_type: 
-          - "Biospecimen"
-          - "Patient"
-
-    model:
-        input:
-            location: "data/schema_org_schemas/example.jsonld" # path to JSON-LD data model
-            file_type: "local" # only type "local" is supported currently
-    style: # configuration of google sheet
-        google_manifest:
-          req_bg_color:
-            red: 0.9215
-            green: 0.9725
-            blue: 0.9803
-          opt_bg_color:
-            red: 1.0
-            green: 1.0
-            blue: 0.9019
-          master_template_id: '1LYS5qE4nV9jzcYw5sXwCza25slDfRA1CIg3cs-hCdpU'
-          strict_validation: true
+There are some defaults in schematic that cna be configured. These fields are in ``config_example.yml``. If you want to change any of these copy ``config_example.yml`` to ``config.yml``, change any fields you want to, and remove any fields you don't.
```

Comment:
Will the code still work if users delete certain keys in the `config.yml`? I think in the past we recommend users not to change the key of the config. I am worried that the functions are more likely to break if structure of config changes.

Reply:
Yes you can see [here](https://github.com/Sage-Bionetworks/schematic/blob/feature-fds-273-coniguration/schematic/configuration/configuration.py#L46) the code will grab the section if it exists, otherwise it will return an empty dict. The dict will be unpacked and sent to the appropriate dataclass. The dataclass parameters match the keys in the dict from the yaml so that those that match will become arguments, those that are missing have defaults, and any others will cause an exception.

Reply:
I just noticed a typo. Could you correct that in the Readme? 

And also, I think telling the users that they could "remove" fields in the config.yml is confusing. For example, even if I remove the "manifest" block in the config, as long as I don't input new values, the code is still using the default (which is the same as the config). Removing fields doesn't mean that the code is no longer using the default values.